homeassistant/helpers/config_validation.py

```diff
@@ -690,6 +695,16 @@ def validator(value):
     return validator
 
 
+def custom_serializer(schema):
+    """Serialize additional types for voluptuous_serialize."""
+    import voluptuous_serialize
+
+    if schema == positive_time_period_dict:
```

Comment:
We want to check for pointer equivalency.

```suggestion
    if schema is positive_time_period_dict:
```

Reply:
Fixed.